aiven/client/cli.py

```diff
@@ -5546,6 +5546,77 @@ def service__flink__cancel_application_deployments(self) -> None:
             ),
         )
 
+    @arg.project
+    @arg.service_name
+    def service__custom_file__list(self) -> None:
+        """List all the custom files for the specified service"""
+        self.print_response(
+            self.client.custom_file_list(
+                project=self.args.project,
+                service=self.args.service_name,
+            ),
+        )
+
+    @arg.project
+    @arg.service_name
+    @arg("--file_id", help="A file ID, usually an uuid4 string", required=True)
+    @arg("--target_filepath", help="Where to save the file downloaded")
+    @arg("--stdout_write", help="Write to the stdout instead of file", action="store_true")
```

Comment:
Not sure if this flag makes sense. E.g from the face of it (if I read this correctly) the default if target_filepath is not given is to write to stdout, so the question is what is the use case for getting the file but not writing it to anywhere?

Reply:
I see it like `fetch file | grep foo | sed 's/o/i/' >> new_file.txt`